main/models.py

```diff
@@ -24,3 +24,4 @@ class DruncMessage(models.Model):
     topic = models.CharField(max_length=255)
     timestamp = models.DateTimeField()
     message = models.TextField()
+    severity = models.CharField(max_length=10, default="INFO")
```

Comment:
Would be nice to use [choices](https://docs.djangoproject.com/en/5.1/ref/models/fields/#choices) here to limit the possible values of the field.